src/main/java/com/eerussianguy/blazemap/FMLEventHandler.java

```diff
@@ -11,6 +12,7 @@ public class FMLEventHandler {
     public static void init() {
         final IEventBus bus = FMLJavaModLoadingContext.get().getModEventBus();
 
+        bus.addListener(BlazeMapFeaturesClient::onKeyBindRegister);
```

Comment:
I had thought the `@SubscribeEvent` had registered the event automatically, but turns out I needed to manually register it for it to fire and therefore have the key control config menu recognise it.

Reply:
You could've also added the annotation `@Mod.EventBusHandler` to the `BlazeMapFeaturesClient` class tbf

Reply:
This can stay for now, later we'll review all the event stuff in 1.20